docs/source/advanced.rst

```diff
@@ -25,8 +25,9 @@ Turn Off Private Key
 
 By default, all OnionShare services are protected with a private key, which Tor calls "client authentication".
 
-The Tor Browser will ask you to enter your private key when you load an OnionShare service.
-If you want allow the public to use your service, it's better to disable the private key altogether.
+The Tor Browser will ask you to enter your private key when you want to offer an OnionShare service.
```

Comment:
Is this done for just authentication, or also encryption.

Reply:
Authentication. Tor's onion service protocol itself takes care of encryption.

Reply:
I don't think the proposed change is accurate, actually. Private key is prompted by the browser when visiting (not 'offering') an OnionShare service. The original sentence is correct.